package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "airframe-dashboard",
   "version": "0.1.0",
+  "proxy": "http://localhost:3001",
```

Comment:
What does this do? / Shouldn't this port number be dynamic?

Reply:
After my most recent changes, this proxy will only be used in development to point the client to the backend server.